spec/jobs/request_otp_sms_job_spec.rb

```diff
@@ -2,24 +2,17 @@
 require 'sidekiq/testing'
 
 RSpec.describe RequestOtpSmsJob, type: :job do
-  Sidekiq::Testing.fake!
-
   let!(:user) { create(:user, phone_number: '1234567890') }
-  let!(:sms_notification_service) { double(SmsNotificationService.new(nil, nil)) }
-
-  it 'calls off to the SMSNotificationService to deliver the otp SMS' do
-    expect(SmsNotificationService)
-      .to receive(:new)
-            .with(user.phone_number, ENV['TWILIO_PHONE_NUMBER'])
-            .and_return(sms_notification_service)
+  let(:app_signature) { ENV["SIMPLE_APP_SIGNATURE"] }
```

Comment:
Is it okay for this to be blank? I see its empty in our .env.test file.

Reply:
Yep, perfectly fine for it to be blank.